lib/offsite_payments/integrations/sage_pay_form.rb

```diff
@@ -69,14 +69,20 @@ def cipher(action, key, payload)
       class Helper < OffsitePayments::Helper
         include Encryption
 
+        attr_reader :timestamp
+
+        def initialize(order, account, options={})
+          super
+          @timestamp = Time.now.strftime('%Y%m%d%H%M%S').split(//).last(4).join
```

Comment:
@girasquid That looks cleaner to me, thanks! @andrewpaliga suggested using the last four digits in the original issue, I'm not sure if that was for a particular reason or seemed easiest. I'll let him comment on that and revise as needed.